templates/index.html

```diff
@@ -15,10 +15,22 @@
         <div class="container">
             <div class="col-md-6 col-md-offset-3">
                 <div class="page-header">
-                    <h2>REDCap to Harvest</h2>
-
-                    <p class="lead">Preview and interact with a live Harvest application built directly from your REDCap project!</p>
-                </div>
+		    <div class="row">
```

Comment:
These divs are no longer necessary. Also remove the link to the containers page since that is for operational purposes (us) and not for users. So revert his file to it's previous state.